### DISCUSSION TITLE
labelling leaves with sample names?

### DISCUSSION THREAD

**Discussion by chlangley** [tskit-dev/tskit#2088] [Q&A] [answered]
Is there a direct way to label leaf nodes with sample names?

**Comment by petrelharp** [comment, marked as answer]
Do you mean in the tree sequence data structure itself or when you make a picture for output, e.g., with `.draw_svg( )`?  For the latter we've got [an example here](https://tskit.dev/tutorials/viz.html#labelling). For the former, we could add metadata - that would probably make a good example? (Does tsinfer already do this?)

**Reply by chlangley** [reply]
Sorry, I should be much clearer.y
I have an a simplify inferred tree, as in reduced_tree = inferred_ts.simplify([0,...
using reduced_tree.draw_svg("path2output.svg", ,x_lim=(62543294, 62550000))

**Reply by chlangley** [reply]
That looks quite helpful.  thanks.
